system/lib/libc/musl/src/thread/pthread_getspecific.c

```diff
@@ -4,10 +4,6 @@
 static void *__pthread_getspecific(pthread_key_t k)
 {
 	struct pthread *self = __pthread_self();
-	// XXX EMSCRIPTEN: self->tsd can be NULL in the case of the
-	// main thread where pthread_key_create is not called.
-	// See __pthread_key_create for where it get assigned.
-	if (!self->tsd) return NULL;
```

Comment:
why this change?

Reply:
As @kleisauke pointed out its no longer true that self->tsd can be NULL since we now initialize it on startup of both the main thread and any secondary threads.

Reply:
I could perhaps split this change out..

Reply:
I see, thanks. sgtm as it is.